README.md

```diff
@@ -510,6 +510,13 @@ The actual lookup edit cell is a fork of the one authored by jlyon87 as found [h
 
 </details>
 
+<details id="soql-datatable---custom-column-labels>
```

Comment:
Not sure if the `id` part is killing the secret detail formatting, but this isn't showing up correctly in both my local (markdown preview) and also the github markdown preview.

Can you fix this and make sure it shows up correctly in the README?

I think it's either:

1) The id attribute, or...
2) Not enough line breaks between your body of text and the closing tag (`</details>`).

Reply:
Weird - I previewed it using the VS Code “Open Preview” and it worked fine (including the “scroll to” behavior controlled by the id), but I’ll  check this out

Reply:
Ah, it was the lack of closing quotes at the end of the id. I must have hit ctrl+z one too many times after having done the preview 🤦‍♂️